iocs/xspress3IOC/iocBoot/ioc_4Channel/st.cmd

```diff
@@ -8,7 +8,7 @@ epicsEnvSet("PREFIX", "XSP3_4Chan:")
 epicsEnvSet("NUM_CHANNELS",  "4")            
 
 # Number of xspress3 cards and IP ADDR
-epicsEnvSet("XSP3CARDS", "1")
+epicsEnvSet("XSP3CARDS", "2")
```

Comment:
If we are assuming the mini by default then we also need to change st.cmd for the following example ioc_<N>Channel boot scripts:

- 7 channels
- 14 channels
- 16 channels